README.md

```diff
@@ -23,6 +23,8 @@ Because open-source Kubernetes depends on your specific configuration, Pega does
 
 The helm charts currently only support running on Kubernetes nodes with x86-64 CPUs.  ARM CPUs are currently unsupported.
```

Comment:
I realize this was not changed, but it is not accurate.  It has nothing to do with the Helm Charts, Pega Infinity does not support running on ARM CPUs because it leverages libraries that are not compatible with ARM CPUs.   Once Pega Infinity is updated to support ARM CPUs, no changes are required to the Helm Charts.   This ARM CPU restriction information should be moved to the Platform Support Guide, and shouldn't be mentioned here, we cannot have multiple documentation locations to itemize hardware and software support.

Reply:
I removed the paragraph from the README and will work with Susan Stewart to add it to the Platform Support Guide instead. 
Is the addition about K8s release information correct, or is there anything else you wish to change here? Thanks in advance.

Reply:
Yes, the supported K8S versions statement is accurate.

Reply:
Thank you for the confirmation. Could you please also approve the pull request so I can merge it?